eml/api/api.js

```diff
@@ -40,18 +58,31 @@ export const getCourseWithAuth = async (courseId) => {
   return res.data;
 };
 
-export const getCourses = async () => {
-  // TODO: add bearer token to request header and omit /public
-  const res = await axios.get(url + '/api/public/courses');
-  return res.data;
-};
 
 export const getCourse = async (courseId) => {
-  const res = await axios.get(url + '/api/public/courses/' + courseId);
+  const res = await axios.get(url + '/api/course/' + courseId)
   return res.data;
 };
 
-// TODO: Endpoint for getcoursebyid && change getCourses to getCourseList
+
+
+//CREATE FOR TESTING: BUT IT WORKS
```

Comment:
If this is not used, I think we should remove it completely.

Reply:
removed